src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/ObjectExplorerService.cs

```diff
@@ -393,6 +395,27 @@ internal ExpandResponse QueueExpandNodeRequest(ObjectExplorerSession session, st
         {
             NodeInfo[] nodes = null;
             TreeNode node = session.Root.FindNodeByPath(nodePath);
+
+            // Performance Optimization for table designer to load the database model earlier.
+            if (node.NodeTypeId == NodeTypes.Database)
+            {
+                Task.Run(() =>

```

Comment:
since each of these starts a new Task, it won't slow down the loading of the object explorer, right? Even if there are 100+ dbs?

Reply:
this will only happen if the user expands the database node.

Reply:
reverted